src/libs/Notification/clearReportNotifications/index.native.ts

```diff
@@ -0,0 +1,32 @@
+import Airship, {PushPayload} from '@ua/react-native-airship';
+import Log from '@libs/Log';
+import {NotificationData} from '@libs/Notification/PushNotification/NotificationType';
+import ClearReportNotifications from './types';
+
+function parseNotificationAndReportIDs(pushPayload: PushPayload) {
+    let payload = pushPayload.extras.payload;
+    if (typeof payload === 'string') {
+        payload = JSON.parse(payload);
+    }
```

Comment:
I'm just thinking, but if we end up zipping the payload we'll have to remember to update this too, or will we pass in the already uncompressed payload as a param?

Reply:
Yeah we have this `JSON.parse` in a couple places so we'd have to update all of those. Not sure what Airship's SDK will do but hopefully it passes the payload uncompressed so we don't have to do anything special